### PR TITLE
[TTAHUB-2549] Minimal Default Docker Configuration

### PR DESCRIPTION
**Pull request by GarrettEHill**
## Description of change

- Changed ```yarn docker:start``` to only run the minimal required containers.
- Added ```yarn docker:start:full``` to run all the containers
- switch from ```docker-compose``` to ```docker compose```

## How to test
run the yarn commands

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2549

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

**Review by nvms — APPROVED** [review]
Looks good.

https://github.com/HHS/Head-Start-TTADP/blob/TTAHUB-2549%2Fminimal-required/README.md#L90-L92

I think this will need to be re-worded/removed since we would no longer require `docker-compose`.

Edit: looks like that link doesn't actually take you to those lines... but it's this bit right here:

```markdown
On a Mac with Brew installed Docker, yarn commands may fail due to the absence of `docker-compose` (vs `docker compose`).  To resolve:

`brew install docker-compose` 
```

File: bin/prod-style-server [hunk 1]
```diff
@@ -9,7 +9,7 @@ exec > >(tee -a "setup.log") 2>&1
 
 echo "Starting server setup..."
 
-# Check for existence of required docker-compose file
+# Check for existence of required docker compose file
```
Comment (thewatermethod) [inline bin/prod-style-server:12]: I think find and replace betrayed you here

**Review by thewatermethod — APPROVED** [review]
Works great, awesome change

File: bin/prod-style-server (hunk 1 above)
Comment (GarrettEHill) [reply bin/prod-style-server:12]:
```suggestion
# Check for existence of required docker-compose file
```

**Pull request merged**